presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -231,13 +230,12 @@ public void testTime()
                 assertEquals(rs.getObject(column), someBogusValue); // TODO this should represent TIME '00:39:07 +01:00'
                 assertThatThrownBy(() -> rs.getDate(column))
                         .isInstanceOf(SQLException.class)
-                        // TODO (https://github.com/prestosql/presto/issues/5313) wrong message
                         .hasMessage(serverSupportsVariablePrecisionTimeWithTimeZone()
                                 ? "Invalid date from server: 00:39:07+01:00"
                                 : "Invalid date from server: 00:39:07.000 +01:00");
                 assertEquals(rs.getTime(column), someBogusValue); // TODO this should fail, as there no java.sql.Time representation for TIME '00:39:07' in America/Bahia_Banderas
                 assertThatThrownBy(() -> rs.getTimestamp(column))
-                        .isInstanceOf(IllegalArgumentException.class) // TODO (https://github.com/prestosql/presto/issues/5315) SQLException
+                        .isInstanceOf(SQLException.class) 
```

Comment:
Remove redundant whitespace. 
```suggestion
                        .isInstanceOf(SQLException.class)
```

```bash
2020-11-26T15:30:37.4873074Z [ERROR] src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java:[302] (regexp) RegexpMultiline: Line has trailing whitespace
2020-11-26T15:30:37.4883585Z [ERROR] src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java:[336] (regexp) RegexpMultiline: Line has trailing whitespace
2020-11-26T15:30:37.4893711Z [ERROR] src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java:[351] (regexp) RegexpMultiline: Line has trailing whitespace
2020-11-26T15:30:37.4903756Z [ERROR] src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java:[431] (regexp) RegexpMultiline: Line has trailing whitespace
2020-11-26T15:30:37.4923878Z [ERROR] src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java:[451] (regexp) RegexpMultiline: Line has trailing whitespace
2020-11-26T15:30:37.4934079Z [ERROR] src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java:[464] (regexp) RegexpMultiline: Line has trailing whitespace
```